scripts/quickstart.sh

```diff
@@ -5,6 +5,27 @@
 
 trap 'kill 0' SIGTERM
 
+# Start local object storage, if desired.
+if [ -n "${MINIO_ENABLED}" ]
+then
+  export MINIO_ACCESS_KEY="THANOS"
+  export MINIO_SECRET_KEY="ITSTHANOSTIME"
+  export S3_ACCESS_KEY=${MINIO_ACCESS_KEY}
+  export S3_SECRET_KEY=${MINIO_SECRET_KEY}
+  export S3_BUCKET="thanos"
+  export S3_ENDPOINT="127.0.0.1:9000"
+  rm -rf data/minio
+  mkdir -p data/minio
+
+  minio server ./data/minio \
+      --address ${S3_ENDPOINT} &
+  sleep 3
+  # create the bucket
+  mc config host add tmp http://${S3_ENDPOINT} THANOS ITSTHANOSTIME
```

Comment:
Use `MINIO_*` env vars?

Reply:
For sure, I had assumed that if you were `MINIO_ENABLED`, you'd also set `S3_ENDPOINT` to the Minio url. But that's probably not a valid assumption. I'll probably change the env vars up a bit so that it's easy/obvious to use minio or actual s3 in this script.